flexget/plugins/modify/convert_magnet.py

```diff
@@ -33,8 +33,10 @@ def magnet_to_torrent(self, magnet_uri, destination_folder, timeout):
         import libtorrent
         params = libtorrent.parse_magnet_uri(magnet_uri)
         session = libtorrent.session()
-        # for some reason the info_hash needs to be bytes but it's a struct called sha1_hash
-        params['info_hash'] = bytes(params['info_hash'])
+        lt_version = [int(v) for v in libtorrent.version.split('.')]
+        if lt_version > [0,16,13,0]:
+            # for some reason the info_hash needs to be bytes but it's a struct called sha1_hash
+            params['info_hash'] = params['info_hash'].to_bytes()
```

Comment:
From what I understand, the info_hash value still has to be set for older versions of libtorrent. It just expects a different type.